### ISSUE TITLE
Installation wizard - Node setup test buttons - Cosmos base-chain

### ISSUE THREAD

**Issue by crsfalzon** [SimplyVC/panic#199]
### User Story

As a node operator, I want to be able to verify that the various URLs requested are correctly configured before my settings are finalised.

### Description

The scope of this task is **limited exclusively to the frontend aspect** of test buttons associated with the nodes setup step - see #48.

### Requirements

Introduce a Test button for the following per node added:

- Cosmos rest server
- Prometheus endpoint URL
- Node exporter URL 
- Tendermint RPC URL

### Blocked by

Not dependent on another ticket to be tackled.

### Acceptance criteria

**Scenario**: Node operator is using the installation wizard to setup monitoring and alerting for a Cosmos based chain
**Given**: The node operator is in the Nodes setup step
**Then**: The node operator is presented with a Test button corresponding to Cosmos rest server, Prometheus endoint URL, Node exporter URL, Tendermint RPC URL per node he wants to set up.
